### DISCUSSION TITLE
How to view the inside of the Sponza.gltf?

### DISCUSSION THREAD

**Discussion by cx20** [Reon90/redcube#33] [Q&A] [answered]
I tried to display the [Sponza.gltf](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Sponza) in RedCube.js.
However, I didn't know how to view the inside of the building.

[Expected result:](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Sponza)
![image](https://user-images.githubusercontent.com/1833979/108594895-fe759a80-73bf-11eb-9817-dc8da72d5982.png)

[Current result:](https://cx20.github.io/gltf-test/examples/redcube/index.html?category=tutorialModels&model=Sponza&scale=1&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/108594906-0a615c80-73c0-11eb-9a37-d778a2cbbf95.png)

**Comment by Reon90** [comment, marked as answer]
@cx20 fixed in 2.5.12

**Reply by cx20** [reply]
@Reon90 Thanks! I could see inside the building.

![image](https://user-images.githubusercontent.com/1833979/108716611-b0979880-755f-11eb-951c-a545bcea5695.png)
